datashuttle/tui/app.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+from time import monotonic
+
+from textual.app import App, ComposeResult
+from textual.binding import Binding
+from textual.widgets import (
+    DirectoryTree,
+    Footer,
+    Header,
+    Input,
+    Label,
+    TabbedContent,
+    TabPane,
+)
+
+
+class TuiApp(App):
+    """
+    The main app page for the DataShuttle TUI.
+
+    Running this application in a main block as below
+    if __name__ == __main__:
+         app = MyApp()
+         app.run()
+
+    Initialises the TUI event loop and starts the application.
+    """
+
+    TITLE = "DataShuttle"
+
+    # Set path to TCSS
+    tui_path = Path(__file__).parent
+    CSS_PATH = tui_path / "css" / "tab_content.tcss"
+
+    # Set key-bindings
```

Comment:
Same here, though I'm conflicted because the reader does have to do some inference from the `Binding()` initialisation  to determine it is key bindings specifically. Can the variable name be changed to `KEY_BINDINGS` or is this fixed in textual?

Reply:
Unfortunately this variable name is fixed in textual!

Reply:
Okay that makes sense, thanks!  We can leave the comment here